packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -29,9 +29,8 @@
 }
 
 .header {
-  border-block-end: 1px solid;
+  border-block-end: 1px solid var(--calcite-ui-border-3);
```

Comment:
moved these together since `border-block-end` isnt exactly the same as `border-bottom`. Applies to other CSS